### ISSUE TITLE
Plugin Automation?

### ISSUE THREAD

**Issue by aj203355** [i-net-software/dokuwiki-plugin-siteexport#51]
I'm trying to setup the plugin for automation. I read the README document and it references the three links generated at the bottom of the siteexport page that change based on the options you select. I also noticed the section that mentions the CRON functionality. Could you provide a little clarification on how to use all the mentioned items to automate the plugin?

**Comment by gamma** [comment]
I think the cron section does not really work - I thought I removed the code already ...

The other links can be used to run your own cron task. They represent the URLs and commands for wget , curl or the pure URL access. When calling any of the URLs/bash lines the export will be created and downloaded.

**Comment by aj203355** [comment]
Yeah you removed the cron section. Wasn't aware there ever was cron code
until you mentioned it so yeah its not there. I hate to bother but I've
scoured the Internet and couldn't find any examples of using the cron.php
file for automation. Would it be too much trouble to post an example of
what would go in the cron.php file and how the crontab command would be
constructed to create the cron job?
